core/templates/filters/string-utility-filters/normalize-whitespace-punctuation-and-case.pipe.ts

```diff
@@ -34,6 +34,10 @@ implements PipeTransform {
         return 'qwertyuiopasdfghjklzxcvbnm0123456789'.indexOf(
           character.toLowerCase()) !== -1;
       };
+      let isAdditionalChars = function(character: string) {
+        return '/@$!{}()'.indexOf(
+          character) !== -1;   //more additional characters can be included as per demand
```

Comment:
Can the comment be a bit more descriptive. Put in the comment on the line above the return statement. Comments need a space after `//` and start with a capital letter and end with a full stop. They have to be full sentences.